src/Sema.zig

```diff
@@ -7835,6 +7836,10 @@ fn zirArrayTypeSentinel(sema: *Sema, block: *Block, inst: Zir.Inst.Index) Compil
     const elem_src: LazySrcLoc = .{ .node_offset_array_type_elem = inst_data.src_node };
     const len = try sema.resolveInt(block, len_src, extra.len, Type.usize, "array length must be comptime-known");
     const elem_type = try sema.resolveType(block, elem_src, extra.elem_type);
+    try sema.validateArrayElemType(block, elem_type, elem_src);
+    if (elem_type.zigTypeTag() == .Opaque) {
```

Comment:
Isn't this duplicating `validateArrayElemType`?

Reply:
Yes, I forgot to remove it when I made it into a new function.